source/danger.d.ts

```diff
@@ -273,28 +275,16 @@ export interface GitHubIssue {
 // Subtypes specific to issues
 
 export interface GitHubIssueLabel {
-  /**
-   * The identifying number of this label
-   * @memberOf GitHubIssueLabel
-   */
+  /** The identifying number of this label */
   id: number,
 
-  /**
-   * The URL that links to this label
-   * @memberOf GitHubIssueLabel
-   */
```

Comment:
decided to drop the `memberOf` and `type` cause they're useless in TypeScript world